core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -245,32 +245,31 @@ var TopicsAndSkillsDashboardPage = function() {
   this.filterSkillsByStatus = async function(status) {
     await action.click(
       'Skill Dashboard status filter', skillStatusFilterDropdown);
-    await (
-      await browser.driver.switchTo().activeElement()
-    ).sendKeys(status + '\n');
+    var dropdownOption = element(
+      by.cssContainingText('mat-option .mat-option-text', status));
+    await action.click('Skill status filter option', dropdownOption);
   };
 
   this.filterTopicsByKeyword = async function(keyword) {
     await waitFor.visibilityOf(
       topicFilterKeywordField,
       'Topic Dashboard keyword filter parent taking too long to appear.');
     var filterKeywordInput = topicFilterKeywordField.element(
-      searchFieldLocator);
+      keywordFieldInput);
 
     await action.sendKeys(
       'Topic Dashboard keyword filter', filterKeywordInput, keyword + '\n');
   };
 
   this.filterTopicsByClassroom = async function(keyword) {
-    await waitFor.visibilityOf(
-      topicFilterClassroomField,
-      'Topic Dashboard classroom filter taking too long to appear.');
+    await action.click(
+      'Topic Dashboard classroom filter taking too long to appear.',
+      topicFilterClassroomField);
 
+    var dropdownOption = element(
+      by.cssContainingText('mat-option .mat-option-text', keyword));
```

Comment:
ditto: use `protractor-test-*` class as root of selector chain

Reply:
done

Reply:
done